### ISSUE TITLE
configuration key 'ChargingScheduleAllowedChargingRateUnit' accessibility is wrong, and some item is missing

### ISSUE THREAD

**Issue by JungHeum-Park** [matth-x/MicroOcpp#328]
Hi.
When the server sent the request `GetConfiguration`, `"readonly"` of `ChargingScheduleAllowedChargingRateUnit` in the response `configurationKey` was incorrect.
According to the ocpp1.6 document, the accessibility of 'ChargingScheduleAllowedChargingRateUnit' is R(read only), but the library returns '"readonly": false'.

And looks like some keys are missing.

What file should I modify to correct this?

**Log from octt**
```
Validating config items for feature profile: Core
```
```
INFO
Expected mandatory config item "ConnectorPhaseRotation" not found, or with invalid value
```
```
INFO
Expected mandatory config item "TransactionMessageAttempts" not found, or with invalid value
```
```
INFO
Expected mandatory config item "TransactionMessageRetryInterval" not found, or with invalid value
```
```
INFO
Expected mandatory config item "ChargingScheduleAllowedChargingRateUnit" not found, or with invalid value
```

**Data that server receive**

```

[
  3,
  "3bb55d34-8b3a-4dd7-96aa-06e6eb9deb70",
  {
    "configurationKey": [
      {
        "key": "Cst_PreBootTransactions",
        "readonly": false,
        "value": "false"
      },
      {
        "key": "ConnectionTimeOut",
        "readonly": false,
        "value": "60"
      },
      {
        "key": "MinimumStatusDuration",
        "readonly": false,
        "value": "0"
      },
      {
        "key": "StopTransactionOnInvalidId",
        "readonly": false,
        "value": "false"
      },
      {
        "key": "StopTransactionOnEVSideDisconnect",
        "readonly": false,
        "value": "false"
      },
      {
        "key": "LocalPreAuthorize",
        "readonly": false,
        "value": "false"
      },
      {
        "key": "LocalAuthorizeOffline",
        "readonly": false,
        "value": "true"
      },
      {
        "key": "AllowOfflineTxForUnknownId",
        "readonly": false,
        "value": "true"
      },
      {
        "key": "Cst_SilentOfflineTransactions",
        "readonly": false,
        "value": "false"
      },
      {
        "key": "Cst_AuthorizationTimeout",
        "readonly": false,
        "value": "20"
      },
      {
        "key": "Cst_FreeVendActive",
        "readonly": false,
        "value": "false"
      },
      {
        "key": "Cst_FreeVendIdTag",
        "readonly": false,
        "value": ""
      },
      {
        "key": "Cst_TxStartOnPowerPathClosed",
        "readonly": false,
        "value": "false"
      },
      {
        "key": "HeartbeatInterval",
        "readonly": false,
        "value": "60"
      },
      {
        "key": "LocalAuthListEnabled",
        "readonly": false,
        "value": "true"
      },
      {
        "key": "ResetRetries",
        "readonly": false,
        "value": "2"
      },
      {
        "key": "MeterValuesSampledData",
        "readonly": false,
        "value": "Energy.Active.Import.Register,Power.Active.Import"
      },
      {
        "key": "StopTxnSampledData",
        "readonly": false,
        "value": ""
      },
      {
        "key": "MeterValuesAlignedData",
        "readonly": false,
        "value": "Energy.Active.Import.Register,Power.Active.Import"
      },
      {
        "key": "StopTxnAlignedData",
        "readonly": false,
        "value": ""
      },
      {
        "key": "Cst_MeterValueCacheSize",
        "readonly": false,
        "value": "1"
      },
      {
        "key": "MeterValueSampleInterval",
        "readonly": false,
        "value": "0"
      },
      {
        "key": "ClockAlignedDataInterval",
        "readonly": false,
        "value": "0"
      },
      {
        "key": "Cst_MeterValuesInTxOnly",
        "readonly": false,
        "value": "true"
      },
      {
        "key": "Cst_StopTxnDataCapturePeriodic",
        "readonly": false,
        "value": "false"
      },
      {
        "key": "NumberOfConnectors",
        "readonly": true,
        "value": "1"
      },
      {
        "key": "AuthorizeRemoteTxRequests",
        "readonly": true,
        "value": "false"
      },
      {
        "key": "GetConfigurationMaxKeys",
        "readonly": true,
        "value": "30"
      },
      {
        "key": "UnlockConnectorOnEVSideDisconnect",
        "readonly": true,
        "value": "true"
      },
      {
        "key": "LocalAuthListMaxLength",
        "readonly": true,
        "value": "48"
      },
      {
        "key": "SendLocalListMaxLength",
        "readonly": true,
        "value": "48"
      },
      {
        "key": "ReserveConnectorZeroSupported",
        "readonly": true,
        "value": "true"
      },
      {
        "key": "MeterValuesSampledDataMaxLength",
        "readonly": true,
        "value": "8"
      },
      {
        "key": "StopTxnSampledDataMaxLength",
        "readonly": true,
        "value": "8"
      },
      {
        "key": "MeterValuesAlignedDataMaxLength",
        "readonly": true,
        "value": "8"
      },
      {
        "key": "ChargeProfileMaxStackLevel",
        "readonly": true,
        "value": "8"
      },
      {
        "key": "ChargingScheduleAllowedChargingRateUnit",
        "readonly": false,
        "value": "Power"
      },
      {
        "key": "ChargingScheduleMaxPeriods",
        "readonly": true,
        "value": "24"
      },
      {
        "key": "MaxChargingProfilesInstalled",
        "readonly": true,
        "value": "10"
      },
      {
        "key": "SupportedFeatureProfiles",
        "readonly": true,
        "value": "Core,FirmwareManagement,LocalAuthListManagement,Reservation,SmartCharging,RemoteTrigger"
      }
    ]
  }
]
```

**Comment by JungHeum-Park** [comment]
I find how to change accessibility.

Add `true` as last parameter.
https://github.com/matth-x/MicroOcpp/blob/1bf49421a4641a29ddaf5e3de99027bd93d2b22a/src/MicroOcpp/Model/SmartCharging/SmartChargingService.cpp#L321

**Comment by matth-x** [comment]
Thank you for the report!
